src/extension/features/general/resize-sidebar/index.css

```diff
@@ -0,0 +1,36 @@
+/* ReSharper disable UnknownCssVariable */
+:root {
+  --tk-number-resize-sidebar-original-width: 260;
+  --tk-number-resize-sidebar-resizer-width: 3;
+  --tk-number-resize-sidebar-width: var(--tk-number-resize-sidebar-width-from-code, var(--tk-number-resize-sidebar-original-width));
+}
+/* ReSharper restore UnknownCssVariable */
+
+.sidebar-resizer {
```

Comment:
we'll want to prefix this guy with `tk-` since it's our own element/class -- the others look like overrides for YNAB classes so those obviously should stay the same